### PR TITLE
Avatar upload on Settings Page

### PR DESCRIPTION
**Pull request by poulainv**
1. Add mutation allowing to get signed s3 url to upload file. 
ACCESS_KEY & SECRET should be provided. An IAM user has been created: `tottem-graphql` with all required permissions.

2. Add a basic input file to settings page in order to upload new avatar and store URL to `pictureUrl`. Basic verification are made. Size, file format. 

Note that in top bar `picture` from auth0 user is used. This picture URL from auth0 is not updated for now. So it leads to an avatar inconsistency between the topbar & everywhere else.

### CODE REVIEW

**Comment by poulainv** [comment]
You can try here: https://quiet-vincentpoulain.quiet.now.sh/me/settings

File: src/scenes/Me/Settings/Avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            return
+        }
+
+        const { signedRequest, url } = data.signS3
+
+        const res = await axios.put(signedRequest, file, {
+            headers: {
+                'Content-Type': 'multipart/form-data',
+            },
+        })
```
Comment (deonclem) [inline src/scenes/Me/Settings/Avatar.tsx]: Errors aren't handled here, which can soft lock the user:

image


```suggestion
        let res
        try {
            res = await axios.put(signedRequest, file, {
                headers: {
                    'Content-Type': 'multipart/form-data',
                },
            })
        } catch (e) {
            sendMessage('Unabled to upload your image')
            setLoading(false)
            return
        }
```

After:
image

(feel free to change the error message)

**Comment by deonclem** [comment]
Couldn't try it all the way because I'm probably missing the AWS keys, but LGTM otherwise (apart from above comment)!

File: src/scenes/Me/Settings/Avatar.tsx (hunk 1 above)
Comment (poulainv) [reply src/scenes/Me/Settings/Avatar.tsx]: 👌

**Pull request merged**